src/main/java/io/cdap/cdap/metrics/process/gcp/MonitoringConfig.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.metrics.process.gcp;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * Metrics writer configuration.
+ */
+public class MonitoringConfig {
+
+  private static final long DEFAULT_POLL_INTERVAL = 300;
```

Comment:
nit: include uint in variable name. millisecond? or second?

Reply:
Unused variable, removed.